docs/translation-guide.md

```diff
@@ -14,6 +14,19 @@ Once you’ve made a selection, you can provide suggestions for strings that hav
 
 When translating you may encounter a situation where you have to decide between several ways of saying the same thing. In these situations we refer to the Ubuntu [general translator guide](https://help.launchpad.net/Translations/Guide), and for language specific issues we follow Ubuntu's [team translation guidelines](https://translations.launchpad.net/+groups/ubuntu-translators). Following these guides ensure uniform translations, while also allowing anyone to contribute.
 
+If you don't want to use Rosetta, or want to make a lot a changes at once, you can also translate offline. To do so:
+
+* Go to a project you want to translate
+* Choose your language
+* Click on "Download translation"
+
+You are now able to request an e-mail, which gives you the right link to download the translations. Now:
+
+* Choose your favorite text- or po-editor (e.g. [Poedit] (https://apps.ubuntu.com/cat/applications/raring/poedit/))
```

Comment:
Maybe link to `https://apps.ubuntu.com/cat/applications/poedit/` instead of `https://apps.ubuntu.com/cat/applications/raring/poedit/`?